linera-core/src/unit_tests/test_utils.rs

```diff
@@ -80,7 +80,7 @@ where
 {
     state: WorkerState<S>,
     fault_type: FaultType,
-    notifier: Notifier<Notification>,
+    notifier: Arc<ChannelNotifier<Notification>>,
```

Comment:
Why not wrap the `DashMap` inside the `ChannelNotifier` with an `Arc` and derive `Send` for it right away? 🤔 I think using `DashMap` w/o the `Arc` around it is a foot gun anyway.

Reply:
Maybe that would be better. :thinking: 

But at least the foot gun isn't loaded right now: `ChannelNotifier` is not `Clone`.